src/radio-group/radio-button.tsx

```diff
@@ -83,10 +81,10 @@ export default React.forwardRef(function RadioButton(
               [styles['styled-circle-disabled']]: disabled,
               [styles['styled-circle-readonly']]: readOnly,
             })}
-            strokeWidth={isVisualRefresh ? 12 : 8}
+            strokeWidth={6.5}
```

Comment:
This 6.5 value results in 1.04px stroke when rendering: 16 / 100 * 6.5 = 1.04 (16 is svg size in pixels, 100 is viewBox definition, 6.5 is a value inside this viewbox) 

Fractional values are known to cause smoothing issues.

Is it possible to ask you to revisit this math and make it more pixel perfect in the future?